### PR TITLE
[cli][dispatcher] New CLI commands 'watch', 'unwatch' to watch nodes.

### PR DESCRIPTION
**Pull request by EskoDijk**
The watch command becomes mostly useful with log-level set to 'info' or 'debug'. Methods in OTNS.py are also added such that the Python unit test can try these commands.

To make 'watch' useful (avoid an overload of warnings), the log-level of typical watch messages is set to 'info'.

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -381,6 +383,33 @@ Set simulation title with specified font size.
...
+using the "-log info" command line switch. 
+
+```bash
+> watch 1
+Done
+> watch 3 5 6
+Done
+> watch
+1 3 5 6
+> unwatch 1 3 5
```
Comment (simonlingoogle) [inline cli/README.md:408]: Might be a good idea to allow `unwatch all` command?

File: dispatcher/dispatcher.go [hunk 2]
```diff
@@ -1087,6 +1088,17 @@ func (d *Dispatcher) isWatching(nodeid NodeId) bool {
 	return ok
 }
 
+func (d *Dispatcher) GetWatchingNodes() []NodeId {
+	wn := make([]NodeId, len(d.watchingNodes), len(d.watchingNodes))
```
Comment (simonlingoogle) [inline dispatcher/dispatcher.go:1092]:
```suggestion
	watchingNodeIds := make([]NodeId, len(d.watchingNodes), len(d.watchingNodes))
```
Use more meaningful variable names?

File: cli/README.md (hunk 1 above)
Comment (EskoDijk) [reply cli/README.md:412]: Agree, added in the latest commit f1598b6de00e9eae0a5464498d38995f851cb068

File: dispatcher/dispatcher.go (hunk 2 above)
Comment (EskoDijk) [reply dispatcher/dispatcher.go]: Agree, changed in the latest commit f1598b6de00e9eae0a5464498d38995f851cb068

File: dispatcher/dispatcher.go [hunk 3]
```diff
@@ -1087,6 +1088,17 @@ func (d *Dispatcher) isWatching(nodeid NodeId) bool {
 	return ok
 }
 
+func (d *Dispatcher) GetWatchingNodes() []NodeId {
+	watchingNodeIds := make([]NodeId, len(d.watchingNodes), len(d.watchingNodes))
```
Comment (simonlingoogle) [inline dispatcher/dispatcher.go:1092]: Please fix `Go Lint` failures.

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

**Pull request closed without merging**